### ISSUE TITLE
Code block buttons lack tooltips

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#20856]
### Steps to reproduce

Hover over any code block, and look at the expand/copy buttons

#### What did you expect?

The buttons should have tooltips and `aria-label`s.

#### What happened instead?

There is no textual indication of what the buttons do, which hurts accessibility.

### Operating system

NixOS unstable

### Browser information

Firefox 96.0.2

### URL for webapp

develop.element.io

### Application version

Element version: 0041ad7230a6-react-bb03298e6dad-js-9c242a9ce6d9 Olm version: 3.2.8

### Homeserver

Synapse 1.51.0

### Will you send logs?

No

**Comment by t3chguy** [comment]
I don't think this is a good first issue simply because of how those buttons are injected, making it rather difficult to re-use our existing tooltip'd buttons
